SPECS/ucx/ucx.spec

```diff
@@ -3,47 +3,56 @@
 %bcond_with    cuda
 %bcond_with    gdrcopy
 %bcond_without ib
+#knem disable until MOFED Driver is avialable
+%bcond_with    knem
 %bcond_without rdmacm
 %bcond_with    rocm
 %bcond_with    ugni
+#xpmem disable until MOFED Driver is avialable
 %bcond_with    xpmem
 %bcond_with    vfs
-
-Summary:        UCX is a communication library implementing high-performance messaging
-Name:           ucx
-Version:        1.15.0
-Release:        1%{?dist}
-License:        BSD
+%bcond_with    mad
+%bcond_with    ze
+%bcond_without mlx5
+
+Name: ucx
+Version: 1.18.0
+Release: 3%{?dist}
+Summary: UCX is a communication library implementing high-performance messaging
+Group: System Environment/Libraries
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
-Group:          System Environment/Security
-URL:            http://www.openucx.org
-Source0:        https://github.com/openucx/%{name}/releases/download/v%{version}/ucx-%{version}.tar.gz
+License: BSD
+URL: http://www.openucx.org
+Source: https://github.com/openucx/%{name}/releases/download/v1.18.0-rc1/ucx-1.18.0.tar.gz
 
+BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
+Prefix: %{_prefix}
```

Comment:
Why do we need/want to set these two tags?

Reply:
I agree that they don't appear to be used in the file however they are part of the original source file, keeping the files more in sync with the source.